### PR TITLE
Feature/publishing test

### PR DESCRIPTION
**Pull request by StefanoStream**
Preparing v0.3

### CODE REVIEW

File: playback-sdk-android/build.gradle.kts [hunk 1]
```diff
@@ -42,16 +56,40 @@ android {
...
+    targetCompatibility = JavaVersion.VERSION_11
+}
+
+publishing {
+    publications {
+        register("release", MavenPublication::class) {
+//            from(components["release"])
+            groupId = "com.streamamg"
+            artifactId = "playback-sdk-android"
+            version = "0.2"
```
Comment (oleksiiSazhko-pamedia) [inline playback-sdk-android/build.gradle.kts:85]: Should the version number be 0.3?

**Review by oleksiiSazhko-pamedia — APPROVED** [review]

File: playback-sdk-android/build.gradle.kts (hunk 1 above)
Comment (StefanoStream) [reply playback-sdk-android/build.gradle.kts]: Fixed! Thanks @oleksiiSazhko-pamedia

File: playback-sdk-android/build.gradle.kts [hunk 2]
```diff
@@ -42,16 +56,40 @@ android {
...
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(11)
     }
 }
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
```
[the remark this answers is not in the document]
Comment (StefanoStream) [reply playback-sdk-android/build.gradle.kts:76]: Thanks @artem-y-pamediagroup . I'm going to change the java version there

**Comment by StefanoStream** [comment]
> Do we need to also update the Installation section in README?

@artem-y-pamediagroup good point! I've updated the README. Let me know if you can approve.

**Pull request merged**